src/pages/mesh/advanced/index.md

```diff
@@ -614,3 +624,121 @@ aio api-mesh:source:install "AEM Assets API"@0.0.1
 ```bash
 Successfully updated the mesh with the id: MESH_ID
 ```
+
+## aio api-mesh:log-list
+
+The `log-list` command lists the most recent requests for your mesh by rayID. By default, the command shows the 15 most recent requests.
+
+### Usage
+
+```bash
+aio api-mesh:log-list [-i] [--filename <value>] [--help]
+```
+
+### Flags
+
+The following arguments are optional:
+
+`--filename` allows you to download a CSV list of rayIDs with the specified filename. This flag exports all requests from the last 15 minutes.
+
+`-i` or `--ignoreCache` ignores the cached organization, project, and workspace, which allows you to make new selections. You can also manually [modify the cache](../basic/work-with-mesh.md#projects-and-workspaces).
+
+`--help` provides information on the specified command.
+
+### Response
+
+```terminal
+Selected organization: <ORG_NAME>
+Selected project: <PROJECT_NAME>
+Select workspace: <WORKSPACE_NAME>
+ Ray id           Timestamp      Response status Level          
+ ──────────────── ────────────── ─────────────── ────────────── 
+ 1a123456789abcd1 1724766278577  200             log            
+ 1a123456789abcd2 1724766287188  200             log            
+ 1a123456789abcd3 1724766286997  200             log            
+ 1a123456789abcd4 1724766280810  200             error
+ ```
+
+## aio api-mesh:log-get
+
+After finding the desired rayID with the [`aio api-mesh:log-list` command](#aio-api-meshlog-list), you can use the following command to retrieve the logs for a specific rayID:
+
+### Usage
+
+```bash
+aio api-mesh:log-get [RAYID] [-i] [--help]
```

Comment:
Is RAYID optional?  In my mind, it should be <RAYID> since it's not a literal, but the official usage statement is the final word.

Reply:
The official usage statement is not yet available, as the CLI is not yet public, but I will verify the Logging commands once I have access.